docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,121 @@
+---
+Order: 13
+Area: cpp
+TOCTitle: Configure IntelliSense
+ContentId: 381b7ce1-5766-49b0-ad26-f9eedae70e63
+PageTitle: Configure IntelliSense
+DateApproved: 8/23/2023
+MetaDescription: Configure Visual Studio Code IntelliSense in the C/C++ Extension
+
+---
+# Configuring IntelliSense
+
+This article is about configuring the C/C++ extension to provide proper [IntelliSense](/docs/editor/intellisense.md). IntelliSense is a helpful tool built into VS Code that provides various code editing features that help you code faster and more efficiently. For example, code completion, parameter info, syntax highlighting, code actions (light bulbs), and member lists are all generated using IntelliSense.
+
+IntelliSense is powered by a language service, which provides C++ specific language completions in your code. For the extension to provide correct IntelliSense suggestions, you need to configure the C/C++ extension. There are three types of sources that can be used for configuration:
+
+1. A C/C++ compiler
```

Comment:
Is this rendered as an unordered list? Just want to make sure this isn't misinterpreted as being in priority order.

Reply:
Good point - I switched to bullets so users won't make that assumption.